### DISCUSSION TITLE
Minified React error # 294

### DISCUSSION THREAD

**Discussion by SleeplessByte** [react-static/react-static#1435] [General] [answered]
> Imported from [Spectrum]()

#### error "Mini fied React error # 294"
@sozdayka _May 6, 2020 at 6:20pm_

Hi everybody, i have got this error:
```text
Exporting HTML...
Error: Failed exporting HTML for URL 404 (/Users/user/Documents/project/src/pages/NotFound):
  Mini fied React error #294; visit https://reactjs.org/docs/error-decoder.html?invariant=294 
  for the full message or use the non-minified dev environment for full errors and additional 
  helpful warnings.
```

I will be very grateful for decisions or ideas on how to solve the error "Mini fied React error # 294"

**Comment by SleeplessByte** [comment, marked as answer]
@warloque responded with [this spectrum issue](https://spectrum.chat/react-static/general/reactdomserver-does-not-yet-support-suspense~b7ef2733-b73f-46c8-9c2b-5f22cef6af7f), which includes the solution.

> UPD: having spent quite a few hours on this it turned out that missing files in locale directories like `/pages/en/page/404.js` has been the cause of **Suspense** -related build error.

**Comment by jovylle** [comment]
I got this also.

**Reply by SleeplessByte** [reply]
On which page? Have you checked the accepted answer?
